rumqttd/src/server/broker.rs

```diff
@@ -129,6 +130,11 @@ impl Broker {
         Ok(link)
     }
 
+    pub fn alerts(&self, filter: Filter) -> Result<alerts::AlertsLink, alerts::LinkError> {
```

Comment:
documentation so that a user knows what type of filters are allowed
/// alerts/<alert_kind>/<connection_id>
    /// alert_kind belongs to 
    /// - "connect"
    /// - "disconnect"
    /// - "subscribe"
    /// - "unsubscribe"